Python/import.c

```diff
@@ -2784,6 +2785,14 @@ _imp_is_lazy_import_impl(PyObject *module, PyObject *dict, PyObject *key)
     Py_RETURN_FALSE;
 }
 
+static PyObject *
+_imp_set_lazy_imports_impl(PyObject *module)
+{
+    // enable lazy imports
```

Comment:
nit: the comment here is redundant (the next function call literally contains the same words 😀)